CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
 - Add support for Data Repository Associations when using PERSISTENT_2 as DeploymentType for a managed FSx for Lustre.
 - Add `Scheduling/SlurmSettings/Database/DatabaseName` parameter to allow users to specify a custom name for the database on the database server to be used for Slurm accounting.
 - Add head node alarms to monitor EC2 health checks, CPU utilization and the overall status of the head node.
+- Add the option to use EFS storage instead of NFS exports from the head node root volume for intra-cluster shared ParallelCluster, Intel, Slurm, and Login Node data.
```

Comment:
Add `HeadNode/InternalSharedStorageType` to use Amazon EFS as storage ...

Reply:
Login Node should have the same naming convention of head node, so everything lowercase, plus a final "s" because we can have more than 1 login node.

Reply:
Yes but Login Node data is plural so you don’t use an ‘s’ on ‘node’ in this instance. Ill make it lower case!